google/cloud/bigtable/data/_async/client.py

```diff
@@ -393,8 +398,12 @@ def __init__(
         table_id: str,
         app_profile_id: str | None = None,
         *,
-        default_operation_timeout: float = 600,
-        default_per_request_timeout: float | None = None,
+        default_read_rows_operation_timeout: float = 600,
+        default_read_rows_attempt_timeout: float | None = 20,
+        default_mutate_rows_operation_timeout: float = 600,
+        default_mutate_rows_attempt_timeout: float | None = 60,
+        default_operation_timeout: float = 60,
+        default_attempt_timeout: float | None = 20,
```

Comment:
same as above, I think the pydoc is missing most of these

Reply:
Good catch, fixed